jscomp/test/mt.mli

```diff
@@ -1,6 +1,8 @@
 type  eq =
   | Eq :  'a *'a  ->  eq
   | Neq : 'a * 'a ->  eq
+  | StrictEq :  'a *'a  ->  eq
```

Comment:
in the future, when we add new ones, maybe just append in the end, to make diff less